build.gradle

```diff
@@ -346,7 +346,7 @@ dependencies {
   implementation group: 'org.springframework.security', name: 'spring-security-core', version: '5.7.5'
 //  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-client'
 //  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-resource-server'
-
+  implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310'
```

Comment:
you missed the version? think thats why it didnt worked

Reply:
Not needed any longer as I am sending dateTime as string and parsing it in the mapper before saving it to CCD.